sdk/core/Azure.Core/Azure.Core.All.sln

```diff
@@ -93,18 +99,42 @@ Global
 		Release|x86 = Release|x86
```

Comment:
What caused this file to change?

Reply:
Running a script. Looks like we haven't updated this file for some time.